addons/website/views/website_views.xml

```diff
@@ -387,6 +387,38 @@ action = {
             <field name="sequence">0</field>
         </record>
 
+        <record id="theme_view_form_preview" model="ir.ui.view">
+            <field name="name">website.form</field>
+            <field name="model">ir.module.module</field>
+            <field name="mode">primary</field>
+            <field name="arch" type="xml">
+                <form create="false" edit="false" delete="0" js_class="theme_preview_form">
+                  <div class="preview_frame h-100">
+                        <field name='url' widget='iframe'/>
+                        <img alt='' class='img_mobile' style='display:none' src="/website/static/src/img/phone.png"/>
+                  </div>
+
+                </form>
+            </field>
+        </record>
+
+        <record id="theme_install_kanban_action" model="ir.actions.act_window">
```

Comment:
Is not this supposed to replace an existing action ? I don't see a removal

Reply:
well spoted, wrong rebase with your branch...